src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -57,6 +57,12 @@ public class DataAccessRequestService implements ConsentLogger {
   protected static final Timestamp MINIMUM_SUBMITTED_DATE_FOR_DAR_EXPIRATIONS = Timestamp.from(
       Instant.ofEpochSecond(
           LocalDate.of(2024, 9, 30).toEpochSecond(LocalTime.of(0, 0, 0, 0), ZoneOffset.UTC)));
+  private static final String MEMBER = "member";
+  private static final String MEMBERS = MEMBER + "s: ";
+  public static final String ALL_LISTED_PERSONNEL_MUST_SHARE_THE_SAME_INSTITUTION =
+      "All listed personnel must share the same institutional affiliation.  The following list of roles and members must have email addresses associated with your institution: ";
```

Comment:
minor legibility question - can we use a text block to make this a little easier to scan in code?